tasks-by-users/Rehan-Khan-Engineer/02-nested-menu-with-flexbox/style.css

```diff
@@ -0,0 +1,95 @@
+  background-repeat: no-repeat;
+  background-size: 65px;
+  background-position: center;
+}
+
+.social a {
+  text-indent: -9999px;
```

Comment:
what does it do `text-indent` ?

Reply:
It specifies the indentation of the text.( gives some space +ve or -ve to the text from left side). However i used it so that the text disappears from the element.(used font 0 as well). Note: this topic is not covered yet

Reply:
> It specifies the indentation of the text.( gives some space +ve or -ve to the text from left side). However i used it so that the text disappears from the element.(used font 0 as well). Note: this topic is not covered yet

this property specifies the indentation of the first line in a text-block. I'm wondering why did we use `-999px`. what if we ignore this ? What would break ?

Reply:
if you ignore it here, fb and tw elements would still have text in it. text wont disappear.    About the no.9999px, dont know much but i think it has something to do with screen space. if the number was smaller text would appear somewhere on left side of the elenmnt